data-node/src/main/java/org/graylog/datanode/Configuration.java

```diff
@@ -83,6 +83,9 @@ public class Configuration extends BaseConfiguration {
     @Parameter(value = "opensearch_config_location")
     private String opensearchConfigLocation = "config";
 
+    @Parameter(value = "datanode_config_location")
```

Comment:
Please don't start the property name with datanode_ prefix. We are already using it for the env properties, where this would be GRAYOG_DATANODE_DATANODE_CONFIG_LOCATION. Could we go with the `config_dir`, as mentioned in this diagram? https://github.com/Graylog2/search-team-space/blob/main/datanode/datanode-directory-structure.md